bin/feeder_geotif.rb

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env ruby
+ENV['BUNDLE_GEMFILE'] = File.join(File.expand_path('../..', __FILE__), 'Gemfile')
+require 'bundler/setup'
+require 'fileutils'
+require_relative '../lib/processing_framework'
+
+class FeederGeotifClamp <  ProcessingFramework::CommandLineHelper
+  banner 'This tool takes MODIS and VIIRS data and makes geotifs intended for feeder.'
+  default_config 'feeder_geotif'
+
+  option ['-m', '--mode'], 'mode', 'The mode to use.', default: 'default'
```

Comment:
should this have a default?  It's going to try to pull out a set of configs that don't exist.  

Would probably be better if it was a required param and generated an error if it wasn't specified.